### PR TITLE
Basic arithmetic functions and unit tests.

### PR DESCRIPTION
**Pull request by cqliu1**
Functions:
- add
- subtract
- multiply
- divide
- sum
- mod
- abs
- round
- ceil
- floor
- fix

### CODE REVIEW

File: test/functions/abs.spec.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import chai from 'chai';
```
Comment (rashidkpc) [inline test/functions/abs.spec.js]: Replace this line with 

```
import { expect } from 'chai';
```

File: test/functions/abs.spec.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import { abs } from '../../src/functions/abs.js';
+
+chai.expect();
+const expect = chai.expect;
```
Comment (rashidkpc) [inline test/functions/abs.spec.js]: Remove this line and the one above it

**Comment by rashidkpc** [comment]
I made a comment on abs.spec.js, but it applies to all other spec files. Otherwise this looks good.

**Comment by cqliu1** [comment]
I made the changes to all the test files. Take a look when you get a chance.

**Pull request merged**